### PR TITLE
Prepare release v1.9.0

### PR DESCRIPTION
**Pull request by lbajolet-hashicorp**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,19 +14,56 @@
...
+
 * Remove provisioner plugins for Chef, Converge, Puppet, Salt, and Inspec as
      vendored plugins. These plugins have been previously archived and not
      updated in release since being archived. These plugins can be installed
      using `packer init` or with the Packer plugins sub-command `packer plugins install github.com/hashicorp/chef`.
      [GH-12374](https://github.com/hashicorp/packer/pull/12374)
 
+* Removed most of the community plugins from the bundled plugins:
+    When writing a template, users generally include components that are defined
+    by external plugins. To import them, there exists a `packer plugin` command that
```
[the remark this answers is not in the document]
Comment (lbajolet-hashicorp) [reply CHANGELOG.md:26]: Heh of course I done goofed on this one. Good catch

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,19 +14,56 @@
      [GH-12172](https://github.com/hashicorp/packer/pull/12172)
 
 ### PLUGINS
+
 * Remove provisioner plugins for Chef, Converge, Puppet, Salt, and Inspec as
      vendored plugins. These plugins have been previously archived and not
      updated in release since being archived. These plugins can be installed
      using `packer init` or with the Packer plugins sub-command `packer plugins install github.com/hashicorp/chef`.
      [GH-12374](https://github.com/hashicorp/packer/pull/12374)
```
[the remark this answers is not in the document]
Comment (lbajolet-hashicorp) [reply CHANGELOG.md:23]: We can indeed call this out from the `Notes` section, or we could add a `**Breaking Change**` header on those plugin changes? It'll make it clearer that it only impacts plugins if we do that I think. But I don't mind moving those to `Notes` as well if you think it's better.

[the remark this answers is not in the document]
Comment (nywilken) [reply CHANGELOG.md:23]: Our convention is to list breaking changes in the notes section.

[the remark this answers is not in the document]
Comment (lbajolet-hashicorp) [reply CHANGELOG.md:23]: Sounds good then, let's move those to the `Notes` section

**Review by nywilken — APPROVED** [review]
Given our discussion internally about the reword for the breaking change I am approving so you can merge once you have the change in place.

**Pull request merged**